### PR TITLE
[MRG] Update changelog for upcoming 1.3.1 release

### PR DESCRIPTION
**Pull request by redapple**

### CODE REVIEW

File: docs/news.rst [hunk 1]
```diff
@@ -3,6 +3,42 @@
...
+
+New features
+~~~~~~~~~~~~
+
+- Support ``'True'`` and ``'False'`` string values for boolean settings (:issue:`2519`);
+  you can now do something like ``scrapy crawl myspider -s REDIRECT_ENABLED=False``.
+- Support kwargs with ``response.xpath()`` to use :ref:`XPath variables <topics-selectors-xpath-variables>`
+  and ad-hoc namespaces declarations ;
+  this requires at least Parsel v1.1 (:issue:`2457`).
+- Add support for Python 3.6 (:issue:`2485`).
```
Comment (kmike) [inline docs/news.rst]: It seems the linked ticket is more about running tests in Python 3.6; there are still unresolved issues like https://github.com/scrapy/scrapy/issues/2511

Comment (redapple) [inline docs/news.rst]: Correct @kmike . do you know of any other issue with Python 3.6?

Comment (kmike) [inline docs/news.rst]: No.

Comment (rolando) [inline docs/news.rst]: #2509 (fixes #2511) is ready for review.

**Pull request merged**